### ISSUE TITLE
Profile Home multiple OSes

### ISSUE THREAD

**Issue by FintanH** [radicle-dev/heartwood#63]
The [`profile::home`](https://github.com/radicle-dev/heartwood/blob/5f4fc8236ee08ae279a1e09f033bfb5acddcf607/radicle/src/profile.rs#L95) function relies on `HOME` as a fallback for the base directory. It seems like that would fail on Windows, so I wonder if we should use something like [`directories::BaseDirs::home_dir`](https://docs.rs/directories/4.0.1/directories/struct.BaseDirs.html#method.home_dir) instead -- or at least crib the code.

**Comment by cloudhead** [comment]
Yup, though I would say windows support is very low priority right now :D 

But once we need it, that's definitely one of the first things that will break. Will tag the issue so we can keep track.

**Comment by FintanH** [comment]
> Yup, though I would say windows support is very low priority right now :D

Sure :) Just thought about it as I was reworking the RIP on storage so I said I'd keep track of it :)
